pkg/context/helpers.go

```diff
@@ -107,12 +112,14 @@ func (c *TestContext) AddTestDirectory(srcDir, destDir string) {
 	}
 }
 
+// GetOutput returns all text printed to stdout.
 func (c *TestContext) GetOutput() string {
-	return string(c.output.Bytes())
+	return string(c.capturedOut.Bytes())
```

Comment:
Yeah it was a pain that we would get different output depending on if we had --debug on or not, but since we send debug to stderr, this should help our testing.